notebooks/jacobians_hessians.ipynb

```diff
@@ -7,12 +7,22 @@
    "source": [
     "# Jacobians, hessians, and more: composing functorch transforms\n",
     "\n",
+    "<a href=\"https://colab.research.google.com/github/pytorch/functorch/blob/main/notebooks/jacobians_hessians_colab.ipynb\">\n",
+    "  <img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/>\n",
+    "</a>\n",
+    "\n",
```

Comment:
Question: are jacobian_hessians.ipynb and jacobians_hessians_colab.ipynb copies of each other, or is there some mechanism that deduplicates code between them?

It might not be the best experience if to update a tutorial someone has to update both jacobian_hessians.ipynb and jacobians_hessians_colab.ipynb

Reply:
Currently the code is similar but has differences (example the get perf to compare benchmark, the collaboration restart function, a few clearer variable names).
Thus no easy code dedupe ATM. Agree that would be easier, flip side is updates to these might be few relatively and may need to keep things colab specific regardless.example dim for hessian example is large by a100 in main, vs reduced that for colab or it crashes colab kernel).

Reply:
Let me think about if there's anything easy we can do to dedupe, if not, this seems fine (and I'll file an issue to fix this in the future).

The main reason why we used a100 was because that's what is on the machine I have :P, I wouldn't mind dropping that benchmark number to keep these two notebooks in sync